src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -235,7 +234,6 @@ private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
             AuthenticationRequestParameters authenticationRequestParameters,
             AcquireTokenInteractiveParameters acquireTokenInteractiveParameters)
         {
-            using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
```

Comment:
When this method is called by AcquireTokenInteractiveAsync, it will trigger double logging if the LogEventWrapper is here. How about cases where it is invoked explicitly? How about SignInInteractiveAsync? Won't they now stop logging?

Reply:
these are never called from anywhere other than the `AcquireTokenInteractiveAsync`. I marked them private per our discussion